src/helm/benchmark/metrics/classification_metrics.py

```diff
@@ -0,0 +1,47 @@
+from typing import List, Set
+
+from sklearn.metrics import f1_score
+
+from helm.benchmark.adaptation.request_state import RequestState
+from helm.benchmark.metrics.basic_metrics import normalize_text
+from helm.benchmark.metrics.metric import Metric, MetricName
+from helm.benchmark.metrics.statistic import Stat
+
+
+class ClassificationMetric(Metric):
```

Comment:
Should we call this something different since it only deals with population-level F1 score? Also, might be worth clarifying somewhere that it is distinct from the word-level F1 metric we use in most places?